src/fitnesse/responders/testHistory/TestHistoryResponder.java

```diff
@@ -43,6 +46,8 @@ private Response makeTestHistoryResponse(TestHistory testHistory, Request reques
     page.setNavTemplate("viewNav");
     page.put("viewLocation", request.getResource());
     page.put("testHistory", testHistory);
+    String purgeTimes = context.getProperties().getProperty(ConfigurationParameter.PURGE_TIME.getKey());
+    page.put("purgeTimes", StringUtils.isBlank(purgeTimes) ? new String[] { "0", "7", "30" } : purgeTimes.split(","));
```

Comment:
Maybe it's nicer to check for `null` here instead of blank. I believe it would then be possible to not show the option to purge test history by setting an empty value (but still get the default values by not having the setting at all).
Does that make sense? If you change this please add test cases to verify the behaviour also

Reply:
I see what you mean. Having the possibility to completely remove the option to purge a test history makes sense for me.
Since we are mostly talking about purge options here, should I also change the name of the property to purgeOptions instead of purgeTimes?

Reply:
Sounds like an improvement!

Reply:
I changed the name of the property to purgeOptions. And it is now possible to enable the property but leaving the value empty. All buttons, including the Purge Global option, will not be created.